style.css

```diff
@@ -1,3 +1,9 @@
+:root {
+    --primary-color: rgba(0,0,0,0.5);
+    --overlay-color: rgba(0, 0, 0, 0.938);
```

Comment:
I think it should be better with full opacity on the overlay color... You can see what is beneath the nav bar and having the text in the center causes a bit of text overlap with the navigation lets try how it looks with `--overlay-color: #000`